include/macis/gf/eigsolver.hpp

```diff
@@ -17,20 +17,13 @@
 #pragma once
 #include <Eigen/Core>
 #include <Eigen/Sparse>
+#include <lapack.hh>
 
 namespace macis {
 
-using namespace Eigen;
-
-typedef MatrixXd eigMatD;
-typedef SparseMatrix<double, RowMajor> SpMatD;
-
-extern "C" {
-extern int dsteqr_(char *, int *, double *, double *, double *, int *, double *,
-                   int *);
-extern int dsyev_(char *, char *, int *, double *, int *, double *, double *,
-                  int *, int *);
-}
+typedef Eigen::VectorXd VectorXd;
```

Comment:
Per offline discussion. I would just remove these entirely and be explicit about their usage (which you already are doing in the function prototypes)